apps/studio/src/pages/dashboard/edit/index.tsx

```diff
@@ -118,7 +136,7 @@ const EditPage: NextPageWithLayout = () => {
   }, [isCopied])
 
   const handleEditorChange = (value: any) => {
-    setEditorValue(value)
+    setEditorState(value)
     localStorage.setItem('editorValue', value)
```

Comment:
issue: we should remove this local storage as well, since we are pre-loading the page with the contents from the DB now. This is a playground-specific code since playground doesn't have a DB, and was added to prevent any accidental data loss.

Reply:
agreed